qemu/tests/cfg/hotplug_mem.cfg

```diff
@@ -113,6 +113,14 @@
             mem-path = /mnt/kvm_hugepage
             pre_command = "echo 3 > /proc/sys/vm/drop_caches"
             pre_command_noncritical = yes
+        - backend_memfd:
+            no Host_RHEL.m7 Host_RHEL.m8.u0 Host_RHEL.m8.u1 Host_RHEL.m8.u2
+            backend_mem = memory-backend-memfd
+            setup_hugepages = yes
+            hugetlb_mem = yes
+            only hotplug unplug.pluged_memory
```

Comment:
For rhel7 guest, the unplug is not supported.

Reply:
Thanks, add it for all unplug cases.